rpc/core/blocks_test.go

```diff
@@ -335,8 +340,8 @@ func (indexer mockBlockIndexer) Search(ctx context.Context, _ *query.Query) ([]i
 
 // randomBlocks generates a set of random blocks up to the provided height.
 func randomBlocks(height int64) []*types.Block {
-	blocks := make([]*types.Block, height)
-	for i := int64(0); i < height; i++ {
+	blocks := make([]*types.Block, height+1)
```

Comment:
[note to reviewers]
Here, when we want to generate blocks up to a certain `height`, then we gonna have `height+1` number of blocks